spec/classes/puppet_agent_windows_install_spec.rb

```diff
@@ -7,8 +7,8 @@
       is_expected.to_not contain_class('::puppet_agent::windows::install')
     }
   elsif Puppet.version >= "3.8.0"
-    {'5.1' => {:expect_arch => 'x86', :appdata => 'C:/Document and Settings/All Users/Application Data/Puppetlabs'},
-     '6.1' => {:expect_arch => 'x64', :appdata => 'C:/ProgramData/Puppetlabs'}}.each do |kernelmajversion, values|
+    {'5.1' => {:expect_arch => 'x86', :appdata => 'C:\Document and Settings\All Users\Application Data'},
```

Comment:
Probably not, since we're running these tests on Unix systems and `File::SEPARATOR` should be `/` there.